modules/snooker/src/test/ts/atomic/operate/TransformOperationsTest.ts

```diff
@@ -175,4 +176,33 @@ UnitTest.test('TransformOperationsTest', () => {
       [ enO('a', false), enO('a', false), enO('d', false), enO('f', false) ]
     ], 0);
   })();
+
+  // Test basic changing to header (cell)
+  (() => {
+    const check = (expected: Structs.ElementNew[][], grid: Structs.ElementNew[][], rowIndex: number, colIndex: number) => {
+      const structExpected = mapToStructGrid(expected);
+      const structGrid = mapToStructGrid(grid);
+      const actual = TransformOperations.replaceCell(structGrid, rowIndex, colIndex, comparator, Generators.transform(null, 'td')(TestGenerator()).replaceOrInit);
+      assertGrids(actual, structExpected);
+      clearElements();
+    };
+
+    check([[]], [[]], 0, 0);
+
+    check([
+      [ enE('h(a)_0', true) ]
```

Comment:
It'd be nice to refactor these method names: `enE`, `enO`.

Reply:
Honestly, it'd be nice to redo all these tests in snooker. They are quite bad :cry: